docs/releases/5.2.txt

```diff
@@ -123,7 +123,8 @@ CSRF
 Database backends
 ~~~~~~~~~~~~~~~~~
 
-* ...
+* ``"pool"`` option is now supported in :setting:`OPTIONS` on Oracle Database
```

Comment:
change "on Oracle Database" to "for Oracle Database"

Reply:
Done.